### DISCUSSION TITLE
Software freeze issue

### DISCUSSION THREAD

**Discussion by cmsimon08** [sn4k3/UVtools#189] [Q&A] [answered]
Hi,
I'm currently running 2.9.1 release version (on Window x64). It was working like flawlessly for a few times. Then I start to have this bug after installing print/printer profiles. After installation, every time I click "File" tab on the menu, the whole software freezes without any responses. This happens pretty much every time I run the software now, so I have to shut down software from task manager.
![freeze 1](https://user-images.githubusercontent.com/69887978/115988262-9da84100-a56d-11eb-804e-f3214e3e0e5f.PNG)
In task manager, when this happens, the software is not responding and seem to take high CPU storage while happening.
![freeze 2](https://user-images.githubusercontent.com/69887978/115988329-0ee7f400-a56e-11eb-8ea6-efb578668cae.PNG)

Please let me know if this kind of behavior is something expected or there are any known resolution on this error.

btw, I found this software really useful for my printing customization and would like to thank for the hard work.
Cheers,

**Comment by sn4k3** [comment, marked as answer]
"installing print/printer profiles" its just a file copy to a folder and then there are no more checks on that, only when you open the profile manager...
Clicking "File" also have no action from code, so i guess this is a render problem. What graphic hardware you have? Do you have updated drivers? Another thing to try is reinstall UVtools, but i guess it will not fix, but worth the try.
If problem keeps, try downgrade to 2.9.0

**Reply by cmsimon08** [reply]
thanks for the reply. I've updated graphic driver and now it doesn't freeze anymore.
